### ISSUE TITLE
Bad Gateway error - Command failed: logrotate

### ISSUE THREAD

**Issue by rutvik-rathod** [jc21/nginx-proxy-manager#1215]
I have installed the nginx proxy manager using the instructions

but when I try to login the localip:81
I get a promt as
BAD gateway

**Comment by rutvik-rathod** [comment]
![badGatewarError](https://user-images.githubusercontent.com/60855001/124431344-dcd4e880-dd8d-11eb-81ad-52dfa787b7d1.JPG)

**Comment by danner26** [comment]
Not sure if this is your issue but sometimes during startup, the database is still loading/migrating. This causes it to report bad gateway but if you wait a few minutes, it often works.

If that did not fix the issue pull the logs and attach them here. If you are using docker, you can use `docker ps` to find the container id. Then you can run `docker logs --details CONTAINER_ID`
Otherwise the logs live in the normal place

**Comment by frisodubach** [comment]
From what I've encountered, the bad gateway at login is usually from a database error. Either it's not configured correctly, or it simply doesn't work (which is common on ARM-based architecture). Doing what danner26 said will help find the exact issue though.

**Comment by simuuh** [comment]
yes, this message is annoying, but if you wait a few more minutes to log in, the error message will disappear.

**Comment by PlasmatikSteak** [comment]
I get the "Bad Gateway" error as well. Only just got it after upgrading to version 2.9.5.
`docker logs --details CONTAINER_ID` gives the below result
```
 [7/22/2021] [5:58:25 AM] [Migrate  ] › ℹ  info      Current database version: 20210210154703
 [7/22/2021] [5:58:25 AM] [Setup    ] › ℹ  info      Logrotate Timer initialized
 [7/22/2021] [5:58:25 AM] [Global   ] › ✖  error     Command failed: logrotate /etc/logrotate.d/nginx-proxy-manager
 error: skipping "/data/logs/fallback_access.log" because parent directory has insecure permissions (It's world writable or writable by group which is not "root") Set "su" directive in config file to tell logrotate which user/group should be used for rotation.
 error: skipping "/data/logs/fallback_error.log" because parent directory has insecure permissions (It's world writable or writable by group which is not "root") Set "su" directive in config file to tell logrotate which user/group should be used for rotation.
```

**Comment by EvanGrote** [comment]
Also seeing the same error. Rolling back my docker container container to version `2.9.4` “resolved” the issue for me. Will follow this issue for a more permanent fix.

**Comment by mrneutron42** [comment]
I just tried to install into a Ubuntu 20 VM, per the instructions, and got the same Bad Gateway message.
The instructions clearly need to be updated, because they do not work as published.

I followed the advice of drewbeer:
https://github.com/jc21/nginx-proxy-manager/issues/310#issuecomment-603552819
I manually added the config.json file, and the reference to that file in the docker-compose.yaml file.
Then, I manually deleted the 2 database folders that were created on the initial failed attempts, and ran:
sudo docker-compose down
sudo docker-compose up -d
(I found that without sudo there were permission errors.)

After doing all that, I was able to log into the nginx webpage and change the email address and password.
I have not taken the next steps and added web servers.
